attestation-agent/docker/Dockerfile.keyprovider

```diff
@@ -26,7 +26,13 @@ RUN apt-get update && apt-get install -y \
 	pkg-config
 RUN git clone https://github.com/containers/skopeo $GOPATH/src/github.com/containers/skopeo 
 WORKDIR $GOPATH/src/github.com/containers/skopeo
-RUN git checkout v1.14.1
+# The dependency on skopeo is quite fragile as there are several versions of
```

Comment:
So was v1.14.1 working before and now isn't, or did we just not test it properly before, or are we not sure? I'm trying to work out if this pinned commit (which in theory is as stable as picking a released version?) might cause issues in future too?

Reply:
I simply don't know if it worked before, but it for sure contradicts with the documentation that says that skopeo version should be v1.14.2+.

I've retried several combinations of skopeo from v1.14.x, and also the latest v1.16.0 release, none of them worked for me.

May it cause issues in the future?  I am pretty sure it may and it will, unless we work on tests either on the skopeo side or on our side to ensure we'll keep everything working all the time.  But that's a rather big task that I'm not volunteering myself to work on.

Reply:
I had a hope that `skopeo inspect docker://ghcr.io/confidential-containers/test-container:multi-arch-encrypted` could give me information about the version of skopeo used, but nope.

Definitively we should be building the test image out of code (vs someone build and publish manually). There is the (https://github.com/kata-containers/kata-containers/issues/9360) tracking that work but unfortunately it didn't move forward.